ext/wry_ruby/src/application/window.rs

```diff
@@ -0,0 +1,52 @@
+use magnus::{Error, define_global_function, function};
+use wry::application::dpi::LogicalSize;
+use wry::application::window::WindowBuilder;
+
+use wry::{
+    application::{
+      event::{Event, StartCause, WindowEvent},
+      event_loop::{ControlFlow},
+    },
+  };
+
+pub fn WindoWnew(title: String, width: u32, height: u32, resizable: bool) {
+    let event_loop = wry::application::event_loop::EventLoop::new();
+    let mut window_builder = WindowBuilder::new();
+    window_builder = window_builder.with_title(title.clone());
```

Comment:
hm we prolly dont need to clone here ig

Reply:
I'd assume the Rust side copies it, yeah.